src/modules/playRoom/components/ChatBoxComponents/Notification.js

```diff
@@ -0,0 +1,18 @@
+const Notification = (props) => {
+  if (!props.type) {
```

Comment:
Why you have to check ```props.type```, if you need to validate prop you can use ```prop-types``` module

Reply:
`props.type` is `props.status` of message before I changed. Not type of div.

Reply:
I think If this condition for rendering you just place in one place to return